.github/workflows/build-osp-director-operator.yaml

```diff
@@ -160,7 +160,7 @@ jobs:
       uses: redhat-actions/openshift-tools-installer@v1
       with:
         source: github
-        operator-sdk: '1.13.1'
+        operator-sdk: '1.18.2'
```

Comment:
Also need to install go here (see https://github.com/openstack-k8s-operators/osp-director-operator/pull/558/commits/b3cdc7a648bb55b31423cf8eade64035fea43af6)

Maybe we could use this to pull the version from go.mod - https://github.com/marketplace/actions/go-version-action

Reply:
FYI if you want to test these changes - a while back I added logic to these github actions so that they work with our forks.

Just need to add credentials in https://github.com/dprince/osp-director-operator/settings/secrets/actions:
QUAY_USERNAME
QUAY_PASSWORD
REDHATIO_USERNAME
REDHATIO_PASSWORD

Reply:
@olliewalsh thanks. I rebased this morning. Should be good now. I will configure the secrets...